### ISSUE TITLE
Wrongly read number arguments from properties

### ISSUE THREAD

**Issue by kkoripl** [deepjavalibrary/djl-spring-boot-starter#13]
I'm testing Spring Boot DJL auto-configuration starter (v0.7 from maven), but got into some troubles, when trying to use application.properties to set number arguments:

djl.application-type=OBJECT_DETECTION
djl.input-class=ai.djl.modality.cv.Image
djl.output-class=ai.djl.modality.cv.output.DetectedObjects
djl.model-filter.size=512
djl.model-filter.backbone=mobilenet1.0
**djl.arguments.width=121
djl.arguments.height=121
djl.arguments.threshold=0.5**

Properties of djl.arguments, when using them to SSD can't be parsed by Arguments class, as they're read as Strings. Every each of them, there is no exception - threshold, height, width are doing same mess. Therefore there comes an exception at application startup, which prevents app from running:

`Caused by: java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Double (java.lang.String and java.lang.Double are in module java.base of loader 'bootstrap')
	at ai.djl.mxnet.zoo.cv.objectdetection.SingleShotDetectionModelLoader$FactoryImpl.newInstance(SingleShotDetectionModelLoader.java:122) ~[mxnet-model-zoo-0.6.0.jar:na]
	at ai.djl.repository.zoo.BaseModelLoader.loadModel(BaseModelLoader.java:95) ~[api-0.6.0.jar:na]
	at ai.djl.repository.zoo.ModelZoo.loadModel(ModelZoo.java:162) ~[api-0.6.0.jar:na]
	at ai.djl.spring.configuration.DjlAutoConfiguration.model(DjlAutoConfiguration.java:105) ~[djl-spring-boot-starter-autoconfigure-0.7.jar:0.7]
	at ai.djl.spring.configuration.DjlAutoConfiguration$$EnhancerBySpringCGLIB$$e6288e91.CGLIB$model$0(<generated>) ~[djl-spring-boot-starter-autoconfigure-0.7.jar:0.7]
	at ai.djl.spring.configuration.DjlAutoConfiguration$$EnhancerBySpringCGLIB$$e6288e91$$FastClassBySpringCGLIB$$f9b6fd29.invoke(<generated>) ~[djl-spring-boot-starter-autoconfigure-0.7.jar:0.7]
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244) ~[spring-core-5.2.5.RELEASE.jar:5.2.5.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:331) ~[spring-context-5.2.5.RELEASE.jar:5.2.5.RELEASE]
	at ai.djl.spring.configuration.DjlAutoConfiguration$$EnhancerBySpringCGLIB$$e6288e91.model(<generated>) ~[djl-spring-boot-starter-autoconfigure-0.7.jar:0.7]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:564) ~[na:na]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154) ~[spring-beans-5.2.5.RELEASE.jar:5.2.5.RELEASE]
	... 62 common frames omitted`

**Comment by shapirov103** [comment]
Hello, 

lack of strong typing for the arguments in the DLJ core product is the reason for this issue. There is no clear discoverable metadata to properly arrange conversion of data types. However, it should not mean that users should be stuck with this.  I will think of an approach to inject the arguments programmatically or work with the team to see if there is a better way to deal with it. Stay tuned.

**Comment by shapirov103** [comment]
@kkoripl this issue is fixed in the upcoming 0.9 release of the starter. 

You can try it now with the snapshot approach. To do that (if you are not already have it defined) add the snapshot repo to your maven pom:

```
 <repositories>
        <repository>
            <id>djl.ai</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
        </repository>
    </repositories>
```

And the change the dependency version of the autoconfiguration starter to 0.9-SNAPSHOT.

Please let me know if you have any issues.
